resources/assets/sass/_pages.scss

```diff
@@ -309,4 +309,14 @@
 
 .comment-editor .CodeMirror, .comment-editor .CodeMirror-scroll {
   min-height: 175px;
+}
+
+/* FIXME - Ugly hack to modify the media plugin for TinyMCE */
+.mce-floatpanel[aria-label="Insert/edit media"] {
+	.mce-open {
+		display: none;
+	}
+	.mce-tab:last-of-type {
+		display: none;
```

Comment:
From a quick test not sure this rule to hide the tab is needed, I think the tab hides itself when both:

```
media_alt_source: false,
media_poster: false,
```
Are set.

Only quickly tested in codepen myself though.